Makefile

```diff
@@ -35,14 +35,14 @@ BUILDTAGS := netgo osusergo
 endif
 
 ifneq ($(shell uname -s), Darwin)
-LINT_BUILDTAGS := e2e,netgo,osusergo,seccomp
+LINT_BUILDTAGS := e2e,netgo,osusergo,seccomp,-tools
```

Comment:
Even if I added `//nolint` to tools.go the linter would still bark at me during `make verify` so I used this as a workaround. If  someone could make the `//nolint` tag work instead, I'd be glad.